.github/workflows/build.yml

```diff
@@ -9,9 +9,12 @@ on:
   pull_request:
     branches:
       - master
-
+  workflow_dispatch:
+    
```

Comment:
I think we need to be careful with manually triggering the workflow as I am not sure how it will effect push images stage.

Reply:
OK, removed it